gen/airsupportgen.py

```diff
@@ -192,9 +196,12 @@ def generate(self):
 
                 self.air_support.awacs.append(
                     AwacsInfo(
-                        str(awacs_flight.name),
-                        callsign_for_support_unit(awacs_flight),
-                        freq,
+                        dcsGroupName=str(awacs_flight.name),
+                        callsign=callsign_for_support_unit(awacs_flight),
+                        freq=freq,
+                        depature_location="-",
+                        start_time=timedelta(),
+                        end_time=timedelta(),
```

Comment:
Either the `AwacsInfo` members should not be `Optional` or these should be `None`. I'm leaning toward the latter since there isn't an appropriate value, and `None` lets the presentation layer decide how these should be displayed.